### PR TITLE
Update pint

### PR DESCRIPTION
**Pull request by ongbw**
many bulk changes

### CODE REVIEW

File: _bibliography/pint.bib [hunk 1]
```diff
@@ -678,12 +742,26 @@ @article{MadayTurinici2003
 	year = {2003}
 }
 
+@article {SheenEtAl2003,
+	author = {Sheen, Dongwoo and Sloan, Ian H. and Thom\'{e}e, Vidar},
+	title = {{A parallel method for time discretization of parabolic equations based on {L}aplace transformation and quadrature}},
+	journal = {IMA J. Numer. Anal.},
+	fjournal = {IMA Journal of Numerical Analysis},
```
Comment (danielru) [inline _bibliography/pint.bib]: Could you please remove fjournal and use the full name in journal? Just to be consistent with the other entries.

File: _bibliography/pint.bib [hunk 2]
```diff
@@ -678,12 +742,26 @@ @article{MadayTurinici2003
...
+	volume = {23},
+	year = {2003},
+	number = {2},
+	pages = {269--299},
+	issn = {0272-4979},
```
Comment (danielru) [inline _bibliography/pint.bib]: Please remove issn entry.

File: _bibliography/pint.bib [hunk 3]
```diff
@@ -825,6 +903,19 @@ @inproceedings{StaffRonquist2005
...
+@article {Thome2005,
+	author = {Thom\'{e}e, Vidar},
+	title = {A high order parallel method for time discretization of parabolic type equations based on {L}aplace transformation and quadrature},
+	journal = {Int. J. Numer. Anal. Model.},
+	fjournal = {International Journal of Numerical Analysis and Modeling},
+	volume = {2},
+	year = {2005},
+	number = {1},
+	pages = {85--96},
+	issn = {1705-5105}
```
Comment (danielru) [inline _bibliography/pint.bib]: Please remove issn.

File: _bibliography/pint.bib [hunk 4]
```diff
@@ -825,6 +903,19 @@ @inproceedings{StaffRonquist2005
 	year = {2005}
 }
 
+@article {Thome2005,
+	author = {Thom\'{e}e, Vidar},
+	title = {A high order parallel method for time discretization of parabolic type equations based on {L}aplace transformation and quadrature},
+	journal = {Int. J. Numer. Anal. Model.},
+	fjournal = {International Journal of Numerical Analysis and Modeling},
```
Comment (danielru) [inline _bibliography/pint.bib]: Please remove fjournal entry and use full name in journal.

File: _bibliography/pint.bib [hunk 5]
```diff
@@ -990,7 +1081,8 @@ @inbook{Ulbrich2007
 	pages = {145-168},
 	doi = {10.1137/1.9780898718935.ch7},
 	url = {https://dx.doi.org/10.1137/1.9780898718935.ch7},
-	year = {2007}
+	year = {2007},
+	publisher = {SIAM}
```
Comment (danielru) [inline _bibliography/pint.bib]: article entries do not have the publisher, again to be . consistent with other entries.

File: _bibliography/pint.bib [hunk 6]
```diff
@@ -1291,6 +1383,16 @@ @inproceedings{LepsaSandu2010
 	year = {2010},
 }
 
+@inproceedings{Lai2010,
+	author={Lai, C.~H.},
+	booktitle={{Substructing Techniques and Domain Decomposition Methods}},
+	pages={45--70},
+	series={Computational Science, Engineering \& Technology Series},
+	title={{On Transformation Methods and the Induced Parallel Properties for the Temporal Domain}},
+	url={http://dx.doi.org/10.4203/csets.24.3},
```
Comment (danielru) [inline _bibliography/pint.bib]: In addition to URL, please also add DOI entry.

File: _bibliography/pint.bib [hunk 7]
```diff
@@ -1384,6 +1486,20 @@ @article{ChristliebEtAl2011
 	year = {2011}
 }
 
+@article{DouglasEtAl2011,
+	author = {Douglas, C. and Kim, I. and Lee, H. and Sheen, D.},
+	title = {{Higher-order schemes for the {L}aplace transformation method for parabolic problems}},
+	journal = {Comput. Vis. Sci.},
+	fjournal = {Computing and Visualization in Science},
```
Comment (danielru) [inline _bibliography/pint.bib]: See comments above.

File: _bibliography/pint.bib [hunk 8]
```diff
@@ -2470,6 +2587,15 @@ @article{MinionEtAl2015
...
 }
 
+@inproceedings{OngEtAl2015,
+	author = {Ong, Benjamin and Kwok, Felix and High, Scott},
+	booktitle = {Methods in Science and Engineering XXII},
+	publisher = {Spring--Verlag},
+	series = {Lecture Notes in Computational Science and Engineering},
+	title = {Pipeline Schwarz Waveform Relaxation},
+	year = {2015}}
+
```
Comment (danielru) [inline _bibliography/pint.bib]: Please add URL and DOI.

File: _bibliography/pint.bib (hunk 5 above)
Comment (ongbw) [reply _bibliography/pint.bib]: this was not consistent wrt to existing entries. I went through and made it consistent.

File: _bibliography/pint.bib [hunk 9]
```diff
@@ -1236,6 +1317,17 @@ @article{Wu2009
 % 2010
 %
 
+@article{BlouzaEtAl2010,
+	author = {Blouza, A. and Laurent, B. and Kaber, S.~M.},
+	journal = {Communications in Applied Mathematics and Computational Science},
+	number = {2},
+	pages = {241--263},
+	title = {{Parallel in time algorithms with reduction methods for solving chemical kinetics}},
+	url = {http://dx.doi.org/10.2140/camcos.2010.5.241},
```
[the remark this answers is not in the document]
Comment (ongbw) [reply _bibliography/pint.bib]: this was not consistent in existing entries.  i went through and made it all consistent.

**Comment by ongbw** [comment]
the  @InBook data-type doesn't permit both author and editor.  We should probably stick to @InProceedings to be consistent.  These are the remaining warnings we should fix eventually

Warning--empty school in Howse2017
Warning--empty chapter and pages in HwangMunster2018
Warning--empty chapter and pages in Morton2018
Warning--empty publisher in Morton2018
Warning--can't use both author and editor fields in Ruprecht2017_lncs
Warning--empty publisher in Takami2014_b
Warning--can't use both author and editor fields in Ulbrich2015
Warning--can't use both author and editor fields in VanDerValkEtAl2014

**Pull request merged**